app/controllers/packages_controller.rb

```diff
@@ -0,0 +1,78 @@
+class PackagesController < ApplicationController
+  skip_before_filter :verify_authenticity_token, only: [:save]
+
+  def estimate_request # api endpoint
+      @betsy_shipping = params
```

Comment:
I'm not sure I understand why you are creating an instance variable here for something that's already a globally accessible instance method.